test/test_sacc2.py

```diff
@@ -135,20 +136,6 @@ def test_get_sigma():
     assert np.allclose(sigma, expected)
 
 
-def test_quantity_warning():
-    s = sacc.Sacc()
-    with pytest.warns(UserWarning):
-        s.add_tracer('Misc', 'source_0',
-                     quantity='dummy')
-
-
-def test_data_type_warning():
-    s = sacc.Sacc()
-    s.add_tracer('Misc', 'source_0')
-    with pytest.warns(UserWarning):
-        s.add_data_point('cl_wrong', ('source_0', 'source_0'),
-                         0.1, ell=10.)
-
```

Comment:
How about keeping these passing `warn_empty=True`? Just to keep the coverage, I mean.

Reply:
These were tests for the other warnings, the ones we got rid of about unknown quantities and data types. The new test below checks the `indices` method.